lib/i18n.js

```diff
@@ -109,17 +109,27 @@
     },
 
     TranslateableProperties: Ember.Mixin.create({
+        _translationObserver :  function(sender, key){
```

Comment:
This `key` already has the `'Translation'` suffix since that's the property we're observing.